openssl/src/ssl/mod.rs

```diff
@@ -1814,6 +1815,13 @@ impl SslCipherRef {
             String::from_utf8(CStr::from_ptr(ptr as *const _).to_bytes().to_vec()).unwrap()
         }
     }
+
+    /// Returns the handshake digest of the cipher.
+    #[cfg(all(feature = "v111", ossl111))]
+    pub fn handshake_digest(&self) -> Option<MessageDigest> {
```

Comment:
For what ciphers will this be `None`?

Reply:
I have no idea, but [the function does have such a case](https://github.com/openssl/openssl/blob/master/ssl/ssl_ciph.c#L1964).